lib/mozart_fetcher/http_client.ex

```diff
@@ -36,8 +37,20 @@ defmodule HTTPClient do
     end
   end
 
-  defp set_request_headers("https://fabl.api." <> _),
-    do: [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
+  def process_request_url(endpoint = "https://fabl.api." <> _) do
+    # Remove param that is passed to set FABL ctx-service-env header
+    String.replace(endpoint, ~r"&fabl-ctx-service-env=(test|live)", "")
+  end
+
+  def process_request_url(endpoint) do
+    endpoint
+  end
+
+  defp set_request_headers(endpoint = "https://fabl.api." <> _) do
+    # Default to live to match FABL default behaviour
+    ctx_service_env = if String.match?(endpoint , ~r"fabl-ctx-service-env=test"), do: "test", else: "live"
```

Comment:
As we need to pass this header to Fabl, should we keep this function matcher to target Fabl, but read the environment from the bake script? It seems brittle to pass in, in the config and assume it is set a certain way. We are then removing this query string above and using it as a header here. It maybe suggests a temporary workaround, but the archive isn't a temporary thing?

Reply:
So Sam is suggesting to avoid having to parse environments from the page config enad using the Cosmos environment. 
If that's ok for you we can assist in accessing the Runtime enviornment from Cosmos.